### DISCUSSION TITLE
How to update setting with data type array/json?

### DISCUSSION THREAD

**Discussion by aminulloh2002** [spatie/laravel-settings#87] [Q&A]
heres my code 

        $settings->site_name = ["test" => "hello", " test2"=>"world"];

        $settings->save();
        
        return redirect()->back();
  

it returned an error "expected object,found array"
i also tried to json encode the array and returned another error "expected object, found string"

**Comment by rubenvanassche** [comment]
Hi @aminulloh2002,

This looks like a PHP exception, what's the type of `site_name`?
